armi/reactor/blueprints/tests/test_assemblyBlueprints.py

```diff
@@ -0,0 +1,204 @@
+            op: 10.0
+assemblies:
+    fuel a: &assembly_a
+        specifier: IC
+        blocks: [*block_fuel, *block_fuel]
+        height: [1.0, 1.0]
+        axial mesh points: [1]
+        xs types: [A, A]
+"""
+
+    twoBlockInput_wrongHeights = r"""
+nuclide flags:
+    U: {burn: false, xs: true}
+    ZR: {burn: false, xs: true}
+blocks:
+    fuel: &block_fuel
+        fuel1: &component_fuel_fuel1
+            shape: Hexagon
+            material: UZr
+            Tinput: 600.0
+            Thot: 600.0
+            ip: 0.0
+            mult: 1
+            op: 10.0
+        fuel2: &component_fuel_fuel2
+            shape: Hexagon
+            material: UZr
+            Tinput: 600.0
+            Thot: 600.0
+            ip: 0.0
+            mult: 1
+            op: 10.0
+assemblies:
+    fuel a: &assembly_a
+        specifier: IC
+        blocks: [*block_fuel, *block_fuel]
+        height: [1.0]
+        axial mesh points: [1, 1]
+        xs types: [A, A]
+"""
+
+    twoBlockInput_wrongXSTypes = r"""
+nuclide flags:
+    U: {burn: false, xs: true}
+    ZR: {burn: false, xs: true}
+blocks:
+    fuel: &block_fuel
+        fuel1: &component_fuel_fuel1
+            shape: Hexagon
+            material: UZr
+            Tinput: 600.0
+            Thot: 600.0
+            ip: 0.0
+            mult: 1
+            op: 10.0
+        fuel2: &component_fuel_fuel2
+            shape: Hexagon
+            material: UZr
+            Tinput: 600.0
+            Thot: 600.0
+            ip: 0.0
+            mult: 1
+            op: 10.0
+assemblies:
+    fuel a: &assembly_a
+        specifier: IC
+        blocks: [*block_fuel, *block_fuel]
+        height: [1.0, 1.0]
+        axial mesh points: [1, 1]
+        xs types: [A]
+"""
+
+    twoBlockInput_wrongMatMods = r"""
+nuclide flags:
+    U: {burn: false, xs: true}
+    ZR: {burn: false, xs: true}
+blocks:
+    fuel: &block_fuel
+        fuel1: &component_fuel_fuel1
+            shape: Hexagon
+            material: UZr
+            Tinput: 600.0
+            Thot: 600.0
+            ip: 0.0
+            mult: 1
+            op: 10.0
+        fuel2: &component_fuel_fuel2
+            shape: Hexagon
+            material: UZr
+            Tinput: 600.0
+            Thot: 600.0
+            ip: 0.0
+            mult: 1
+            op: 10.0
+assemblies:
+    fuel a: &assembly_a
+        specifier: IC
+        blocks: [*block_fuel, *block_fuel]
+        height: [1.0, 1.0]
+        axial mesh points: [1, 1]
+        xs types: [A, A]
+        material modifications:
+            U235_wt_frac: [0.5]
+"""
+
+    def loadCustomAssembly(self, assemblyInput):
+        yamlString = assemblyInput
+        design = blueprints.Blueprints.load(yamlString)
+        design._prepConstruction(settings.getMasterCs())
+        return design.assemblies["fuel a"]
+
+    def test_checkParamConsistency(self):
+        # make sure a good example doesn't error
+        a = self.loadCustomAssembly(self.twoBlockInput_correct)
```

Comment:
this should technically have an assert statement afterwards to show not only that it runs but also that it loads values as expected

Reply:
Ah, good catch!

I agree.  Pull the most important things out of that assembly and make sure they are correct.